.github/workflows/pr-checks.yml

```diff
@@ -24,22 +24,22 @@ jobs:
           deno-version: v1.x
 
       - name: Lint
-        # Intentionally only on Linux because output will be the same on all 
+        # Intentionally only on Linux because output will be the same on all
```

Comment:
I can't tell why these lines are updated. Is their line ending changed to `\r\n`?

Reply:
Weird I did a regex search and only see `\n` here. I don't remember accidentally touching this line